app/middleware/validators/requestParamsValidator.js

```diff
@@ -0,0 +1,76 @@
+const Joi = require('joi')
+const ValidationException = require('../exceptions/validationException')
+const moment = require('moment')
+
+const validate = (dateFrom, dateTo, isTotalAmountRequired, wereDangerousMeteorsRequired) => {

```

Comment:
please split this logic
validator - middleware that can work with any schema
validation schemas - files with validators for different  endpoints